### PR TITLE
Refactor README

### PR DESCRIPTION
**Pull request by GavinL2001**
- Added prerequisite setup instructions for Linux, Windows, and macOS
- Cleaned up some language clarity

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,24 +2,84 @@
...
+- Docker
+- Avahi running on host system
+- usbmuxd **NOT** running on host system
 
-The only dependency is docker, but it is important to note two things:
+## Installation
 
-- You might not be able to refresh apps if **usbmuxd** is running in the _host_ system, the docker guest already includes it and binds to the usb interface. Make sure your host system is **NOT** running it.
-- You might need to be running the **avahi-daemon** in the host system.
+### Installing Docker (if not installed already)
```
Comment (dreth) [inline README.md:13]: While this is nice to have, I would prefer to just redirect people to the docker website so they can install it themselves, do you mnd removing this? the rest is great, really appreciate your help!

Comment (GavinL2001) [reply README.md:13]: Sure! I personally thought the automated install was a neat thing to have, but I can change it to redirect to their install guide.

**Comment by GavinL2001** [comment]
Changes have been made. Let me know if there’s anything else I should adjust!

File: README.md (hunk 1 above)
Comment (dreth) [reply README.md:13]: I do also like it but I feel like it's out of the scope of the project in some way

File: README.md [hunk 2]
```diff
@@ -2,24 +2,82 @@
...
 
-### docker compose (recommended)
+#### Linux:
```
Comment (dreth) [inline README.md:15]: i like this more, thanks!

**Review by dreth — APPROVED** [review]

**Comment by dreth** [comment]
Thank you Gavin!

**Comment by GavinL2001** [comment]
No problem! I'm happy to help wherever I can! Thanks for making a great project!

**Pull request merged**